lib/LANraragi/Plugin/Metadata/EHentai.pm

```diff
@@ -65,8 +69,8 @@ sub get_tags {
     } else {
         # Craft URL for Text Search on EH if there's no user argument
         ( $gID, $gToken ) =
-          &lookup_gallery( $lrr_info->{archive_title}, $lrr_info->{existing_tags}, 
-                           $lrr_info->{thumbnail_hash}, $lang, $lrr_info->{user_agent}, $domain, $usethumbs);
+          &lookup_gallery( $lrr_info->{archive_title}, $lrr_info->{existing_tags}, $lrr_info->{thumbnail_hash},
+                           $lrr_info->{user_agent}, $domain, $lang, $jpntitle, $additionaltags, $usethumbs, $expunged);
```

Comment:
This change seems to be what breaks the tests.  
`$ua` aka `$lrr_info->{user_agent}` is for some reason not an object reference anymore, haven't looked further.

Reply:
I changed the order of these arguments because the order was weird, but perhaps that order was required?

Reply:
I just checked the code and I have no idea what is wrong here. I will have to leave this one to you.

Reply:
Thought about it a bit more and it's dead simple: The tests in plugin.t call this function directly, so you'll need to change the call in the tests too.

Reply:
Sorry, somehow I thought I did not have this file. The function call has been updated.